tools/formatter/FormatterTool.java

```diff
@@ -96,6 +96,17 @@ public void format(Path path) throws IOException {
     }
   }
 
+  static class BuildFormatter implements BaseFormatter {
+    static BuildFormatter init() {
+      return new BuildFormatter();
+    }
+
+    @Override
+    public void format(Path path) throws IOException {
+      execute("buildifier -mode=fix " + path.toAbsolutePath().toString());
```

Comment:
If we run it as `bazel run //tools:buildifier`, then we don't need to install it, no?

Reply:
@oferb we'll have to run it as `bazel run --direct_run`, I think. Let's try

Reply:
@oferb actually, if we do it like this, we'll have an implicit dependency on having `buildifier` being available as exact Bazel target `//tools:buildifier` -- but `formatter` is self-contained (except for external `yapf`/`clang-format`/`buildifier` deps). Also, `angular/ngcontainer` *already* has `buildifier` available -- so let's not do it 🙂

Reply:
Maybe we can make the dependency explicit using `https://docs.bazel.build/versions/master/be/java.html#java_binary.runtime_deps`?
If we rely on `buildifier` being available, that includes all dev machines, not only `angular/ngcontainer`. Otherwise, devs can't run it the same way.